src/cms/fixtures/test_data.json

```diff
@@ -437,6 +521,42 @@
       "last_updated": "2019-08-12T08:01:42.279Z"
     }
   },
+  {
+    "model": "cms.pagetranslation",
+    "pk": 44,
+    "fields": {
+      "page": 18,
+      "slug": "Gesundheit",
+      "title": "Gesundheit",
+      "status": "PUBLIC",
+      "text": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut ut ante eu nulla mattis luctus. Etiam nec pellentesque tellus, sit amet tristique erat. Donec eu erat ac dui sagittis mollis a vel velit. Donec tincidunt mauris ipsum, vel hendrerit odio pharetra quis. Phasellus sollicitudin risus et vulputate posuere. Nulla sed nisi molestie elit varius posuere. Nulla eu orci eros. Phasellus at hendrerit dolor, et semper leo. Cras auctor enim urna, ut blandit urna elementum quis.",
```

Comment:
I thought, the `text`-field of a pagetranslation should be valid html? In this case, at least `<p>`-tags have to be added... this also applies to all changed `text`-fields.